examples/custom_shaders_uniforms/GlutWindow.cc

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#if __APPLE__
```

Comment:
Do you mind to add Windows support ? like in this file https://github.com/ignitionrobotics/ign-rendering/pull/291/files#diff-cf8db6f4a0d1d2f53ce083ef0efd58225509404c520b85d771454820a2cb73e5